sdk/enter.go

```diff
@@ -318,16 +322,19 @@ func (e *enter) CopyGoogleCreds() error {
 }
 
 // bind mount the repo source tree into the chroot and run a command
+// Called via the multicall interface. Should only have 1 arg which is an
+// enter struct encoded in json.
 func enterChrootHelper(args []string) (err error) {
-	if len(args) < 3 {
-		return fmt.Errorf("got %d args, need at least 3", len(args))
+	if len(args) != 1 {
+		return fmt.Errorf("got %d args, need exactly 1", len(args))
 	}
 
 	e := enter{
-		RepoRoot: args[0],
-		Chroot:   args[1],
-		CmdDir:   args[2],
-		Cmd:      args[3:],
+		RepoRoot: RepoRoot(),
+		Chroot:   filepath.Join(RepoRoot(), "chroot"),
```

Comment:
For the record, this moves these calculations from the parent process to the child.  Seems okay, but is a behavior change.